libraries/bezier-rs/src/subpath/transform.rs

```diff
@@ -78,13 +89,190 @@ impl<ManipulatorGroupId: crate::Identifier> Subpath<ManipulatorGroupId> {
 			(Subpath::new(first_split, false), Some(Subpath::new(second_split, false)))
 		}
 	}
+
+	/// Returns [ManipulatorGroup]s with a reversed winding order.
+	fn reverse_manipulator_groups(manipulator_groups: &[ManipulatorGroup<ManipulatorGroupId>]) -> Vec<ManipulatorGroup<ManipulatorGroupId>> {
+		manipulator_groups
+			.iter()
+			.rev()
+			.map(|group| ManipulatorGroup {
+				anchor: group.anchor,
+				in_handle: group.out_handle,
+				out_handle: group.in_handle,
+				id: ManipulatorGroupId::new(),
+			})
+			.collect::<Vec<ManipulatorGroup<ManipulatorGroupId>>>()
+	}
+
+	/// Returns a [Subpath] with a reversed winding order.
+	pub fn reverse(&self) -> Subpath<ManipulatorGroupId> {
+		Subpath {
+			manipulator_groups: Subpath::reverse_manipulator_groups(&self.manipulator_groups),
+			closed: self.closed,
+		}
+	}
+
+	/// Returns an open [Subpath] that results from trimming the original Subpath between the points corresponding to `t1` and `t2`.
+	/// If `t1` > `t2`, then behavior for computing the resulting Subpath will differ depending on whether the original Subpath is open or closed:
+	/// - If the original Subpath was open, an open subpath from `t2` to `t1` will be returned (with winding in the same direction as the original Subpath).
+	/// - If the original Subpath was closed, an open subpath from `t1` to `t2` that crosses the break between 1 and 0 will be returned (winding in the same direction as the original Subpath from `t1` to `t = 1 = 0` to `t2`).
+	///
+	/// If a path winding in the reverse direction is desired, call `trim` on the `Subpath` returned from `Subpath::reverse`.
```

Comment:
```suggestion
	/// Returns an open [Subpath] that results from trimming the original Subpath between the points corresponding to `t1` and `t2`, maintaining the winding order of the original.
	/// If the original Subpath is closed, the order of arguments does matter.
	/// The resulting Subpath will wind from the given `t1` to `t2`.
	/// That means, if the value of `t1` > `t2`, it will cross the break between endpoints from `t1` to `t = 1 = 0` to `t2`.
	/// If a path winding in the reverse direction is desired, call `trim` on the `Subpath` returned from `Subpath::reverse`.
```